simple-facebook-instant-articles.php

```diff
@@ -212,9 +212,11 @@ public function gallery_shortcode( $atts, $content = '' ) {
 		ob_start();
 
 		echo '<figure class="op-slideshow">';
```

Comment:
Code sniffer might complain about `echo` without escape function. Maybe easier to do it outside of php tags?